### ISSUE TITLE
Does not convert turtle to n-triples

### ISSUE THREAD

**Issue by inf3rno** [njh/easyrdf#314]
I copy-pasted a simple example:

```turtle
@prefix : <http://api.stardog.com/> .
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix stardog: <tag:stardog:api:> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

:Alice a :Person ;
       :knows :Bob .
```

> Error! $resource should be either IRI, blank-node identifier or EasyRdf_Resource. got empty string 

Looks like RDF isn't that easy after all. :D

**Comment by njh** [comment]
Sorry, I have been unable to re-produce this problem.

Either with version 0.9.0 (http://www.easyrdf.org/converter). Or with the master branch.

When converting to N-Triples, I get the output:
```
<http://api.stardog.com/Alice> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://api.stardog.com/Person> .
<http://api.stardog.com/Alice> <http://api.stardog.com/knows> <http://api.stardog.com/Bob> .
```

Are you able to provide any more details about how to reproduce the error?

**Comment by inf3rno** [comment]
@njh Probably it was fixed since February.

**Comment by inf3rno** [comment]
Nope, it isn't. I used the converter on your site: http://www.easyrdf.org/converter
![easyrdf bug](https://user-images.githubusercontent.com/408911/83704645-f6e44a80-a612-11ea-820d-9fda3183e264.png)

**Comment by inf3rno** [comment]
Looks like it happens because I remove the "or Uri" input, which is "http://njh.me/" by default.

**Comment by njh** [comment]
Ah, thanks.

If you provide some input text, then the URI field is used as a base URI by the parser.
The UI doesn't make this clear and the error message is very unhelpful.

But I think the problem is in the converter harness, rather than the parser itself.
The problem exists in both the www.easyrdf.org converter code:
https://github.com/easyrdf/www.easyrdf.org/blob/master/lib/ConverterController.php#L34

And in the converter example:
https://github.com/njh/easyrdf/blob/master/examples/converter.php#L72
